Modules/Core/Common/include/itkThreadSupport.h

```diff
@@ -22,16 +22,6 @@
 
 #include "itkConfigure.h" // For ITK_USE_WIN32_THREADS, ITK_USE_PTHREADS, etc.
 
-// This implementation uses a routine called SignalObjectAndWait()
-// which is only defined on WinNT 4.0 or greater systems.  We need to
-// define this symbol in order to get the prototype for the
-// routine. This needs to be done before we load any system headers.
-#ifdef ITK_USE_WIN32_THREADS
-#  ifndef _WIN32_WINNT
-#    define _WIN32_WINNT 0x0501 // TBB 4.4 requires WinXP (0x0501 or greater)
```

Comment:
I am not sure about removing this. Have you tried compiling with TBB on Windows?

Reply:
I haven't, nor do I have Windows to try.

From the comment, it seems obsolete... Also the comment starts: "This implementation uses a routine called SignalObjectAndWait()" but I don't actually find that function anywhere in ITK... 

But I know little about Windows development so I defer to others...

Reply:
Does github CI have TBB + Windows?  If not, anyone able to try?

Reply:
In may be in the pixi builds:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/pixi.lock#L890

Reply:
This compiles locally, and multi-threading tests pass.

Reply:
> In may be in the pixi builds: [`master`/pixi.lock#L890](https://github.com/InsightSoftwareConsortium/ITK/blob/master/pixi.lock?rgh-link-date=2024-11-28T13%3A28%3A24Z#L890)

`ITK_USE_TBB` needs to be turned on too.

Reply:
I don't think TBB is turned on:
https://github.com/InsightSoftwareConsortium/ITK/blob/c9419ca6b98dc6328153f69a04a27b99d4a7d27d/pyproject.toml#L31-L32